### ISSUE TITLE
API for Portable to discover available .NET Standard versions

### ISSUE THREAD

**Issue by rchande** [NuGet/Home#1826]
For .NET Standard, we're no longer going to do registration within VS for Portable, so we'll need an API from NuGet that returns the available set of standard versions and the platforms (TFMs?) covered by each one.

Related to https://github.com/NuGet/Home/issues/1725.

**Comment by yishaigalatzer** [comment]
@rchande can we make this ask a bit more specific. Can you suggest an abstract API you are looking for? 

Map from what to what exactly?

**Comment by rchande** [comment]
@yishaigalatzer Portable is going to display a list of versions so the user can pick, so minimally, I would need an API like:

`FrameworkName[] GetDotnetStandardVersions()`

This would return things like {"netstandard", "1.0"} and so forth. If all we need to write into project.json is a framework element indicating netstandard 1.0, that would be sufficient. If there's additional stuff we need to write (packages? supports?) we'll need to be given those as well.

Basically, we want this to be driven by data from NuGet, as opposed to dropped on the box by VS setup.

Additionally, we'd like to list for the user the platforms included in the selected version, so we'd like: 
`FrameworkName[] GetPlatformsIncludedInDotnetStandard(FrameworkName standardVersion)`

Which would return things like "Windows 8.1" or ".NET Framework 4.6" etc.

**Comment by joelverhagen** [comment]
Closed with https://github.com/NuGet/NuGet.Client/pull/221.
